radio/src/gui/128x64/model_special_functions.cpp

```diff
@@ -71,6 +71,36 @@ void onCustomFunctionsFileSelectionMenu(const char * result)
 #endif
 
 #if defined(PCBX7)
+
+void onAdjustGvarSourceLongEnterPress(const char * result)
+{
+  CustomFunctionData * cfn = &g_model.customFn[menuVerticalPosition];
```

Comment:
I know this code is only written for X7, but I think removing the HEADER_LINE is good